bin/commands/run_sql.py

```diff
@@ -0,0 +1,33 @@
+import utils
+
+DESCRIPTION = "Run SQL in running d2-docker container"
+NAME = "run-sql"
```

Comment:
Since we use dump files as backups in WHO instances it would be useful to allow importing data from them with ```pg_restore```. 

See https://github.com/SferaDev/dhis2-backend/commit/70ce784c57f4f3c7606e32efa716bd1e9558d0f1